interview-questions.md

```diff
@@ -9,59 +9,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
For your interview questions, please add more of your experience with the answers. For instance, I recall when I explored the use of the keyword `this`. Tell the story of one of the challenges. Tell your thoughts as well as what you researched. These interview questions are opportunities to practice sharing your thoughts with an interviewer/future clients.